addons/grenades/functions/fnc_throwGrenade.sqf

```diff
@@ -36,17 +36,31 @@ if (local _unit) then {
         playSound3D [_file, objNull, false, getPosASL _projectile, _volume, _pitch, _distance];
     };
 
-    private _flashbangParams = (_config >> QGVAR(flashbang)) call BIS_fnc_getCfgDataArray;
-    private _nBangs = _flashbangParams param [0,0];
-    if (_nBangs > 0) then {
+    if (getNumber (_config >> QGVAR(flashbang)) == 1) then {
+        private _amount = 1;
+        if (isNumber (_config >> QGVAR(flashbangAmount))) then {
+            _amount = getNumber (_config >> QGVAR(flashbangAmount));
```

Comment:
I think this variable name is ambiguous - it could refer to the amount of powder, for example, and thus the strength of the effect. If it's the `n` in `nBangs` you didn't like, can we just do `_bangs` and `flashbangBangs` since it's a getNumber, so it's obvious it is a number?

Reply:
flashbangBangs is fine.